### ISSUE TITLE
ReferenceError: window is not defined

### ISSUE THREAD

**Issue by MilanObrenovic** [vasturiano/three-forcegraph#40]
**Describe the bug**
I installed this library, latest version as of now (`1.41.5`), but getting `ReferenceError: window is not defined` error.

**To Reproduce**
1. Go to https://codesandbox.io/s/step-15-trackball-controls-fixed-xdrgv?from-embed=&file=/src/initializeScene.js
2. This tutorial was originally written by https://dev.to/pahund/adding-trackball-controls-to-a-three-js-scene-with-sprites-48hp
3. Use the same code example from `codesandbox` website example into the existing nextjs project.
4. Run the project
5. See error

**Expected behavior**
A mind map graph to be visible just like in the example

**Screenshots**
<img width="726" alt="image" src="https://user-images.githubusercontent.com/56999235/229382025-1477ebdb-dff0-4b12-83e2-ef9d17c29dd3.png">

**Desktop (please complete the following information):**
 - OS: Mac Ventura 13.3
 - Browser: Chrome
 - Version: 111.0.5563.146

**Comment by MilanObrenovic** [comment]
I think i found a solution. for anyone having similar problems:

TLDR: https://stackoverflow.com/questions/58622615/how-to-import-trackballcontrols-from-three-js-module-using-react

Long version: works fine in normal react app but not nextjs app because nextjs is SSR. In nextjs you need to dynamically import this `three-forcegraph` library (as well as some others if needed) and then use it from inside the `useEffect()`
